homeassistant/scripts/check_config.py

```diff
@@ -21,9 +20,10 @@
 from homeassistant.util import yaml
 from homeassistant.exceptions import HomeAssistantError
 
-REQUIREMENTS = ('colorlog==4.0.2',)
-if system() == 'Windows':  # Ensure colorama installed for colorlog on Windows
-    REQUIREMENTS += ('colorama<=1',)
+try:
+    import colorlog.escape_codes  # noqa # pylint: disable=unused-import
+except ImportError:
+    REQUIREMENTS = ('colorlog==4.0.2',)
```

Comment:
This will confuse our requirement generator script. 

The dev docker image should already have colorlog, so should not install it every time?

Reply:
Indeed, it's not added, but there might also be a problem with how we detect colorlog inside Docker If I read this correctly?

```bash
root@linuxkit-00155d5dc21f:/usr/src/app# pip list | grep colorlog
colorlog                          4.0.2
root@linuxkit-00155d5dc21f:/usr/src/app# hass --script check_config --config /config
INFO:homeassistant.util.package:Attempting install of colorlog==4.0.2
Testing configuration at /config
Failed config
  homeassistant:
    - extra keys not allowed @ data['packages']['yr__pack']
```

> **Another approach would be to make this part of the core requirements... (would prefer this to be honest)**

Reply:
We shouldn't merge this. We're breaking something because some other wrapper or runner doesn't install correctly or keep it around. 

This is not something that should be fixed inside Home Assistant as it works as expected.